src/test/shell/bazel/toolchain_transition_test.sh

```diff
@@ -355,4 +354,29 @@ EOF
   expect_log 'extra_lib: message: extra_lib foo, target_dep: target, tool_dep: exec-alpha'
 }
 
+function test_toolchain_transition_cquery() {
```

Comment:
Do you mind adding a comment about what this seemingly innocuous test case is testing for?

Reply:
Done.